docs/2.9.0/docs/canton/usermanual/connectivity.rst

```diff
@@ -118,7 +118,11 @@ Such a connection can be configured using the ``connect_multi``:
     .. success:: participant3.domains.connect_multi("mydomain", Seq(sequencer1, sequencer2))
     .. assert:: { participant3.health.ping(participant3); true }
 
-In such a setting, if a sequencer node goes down, the participant will round-robin through the available list of sequencers.
+On the read side, the participant will establish a subscription to one available sequencer to subscribe to the stream of
+encrypted messages. If the sequencer fails, the participant will automatically attempt to resubscribe to the next available
+sequencer. On the write path, the participant will send requests to all available sequencers in a round-robin manner, circling
+through the available sequencers in sequence. The same connectivity behaviour applies to the domain manager and the mediator.
```

Comment:
```suggestion
On the read side, the participant will establish a subscription to one available sequencer to subscribe to the stream of
encrypted messages. If the sequencer fails, the participant will automatically attempt to resubscribe to the next available
sequencer. On the write path, the participant will send requests to all available sequencers in a round-robin manner, circling
through the available sequencers. The same connectivity behavior applies to the domain manager and the mediator.
```

Reply:
If it's really important to say that the round-robin happens "in sequence," maybe say "in order" instead. The phrase "available sequencers in sequence" is just not very readable.

Reply:
fixed